samtranslator/internal/schema_source/aws_serverless_function.py

```diff
@@ -333,6 +333,7 @@ class EventBridgeRuleEventProperties(BaseModel):
     Pattern: PassThroughProp = eventbridgeruleeventproperties("Pattern")
     RetryPolicy: Optional[PassThroughProp] = eventbridgeruleeventproperties("RetryPolicy")
     Target: Optional[EventBridgeRuleTarget] = eventbridgeruleeventproperties("Target")
+    InputTransformer: Optional[PassThroughProp] = eventbridgeruleeventproperties("InputTransformer")
```

Comment:
iirc the general guideline was to add a `TODO: add docs` here then update it once the docs are published